.azure_pipelines/olive-ci.yaml

```diff
@@ -39,21 +39,21 @@ jobs:
       bert_ptq_cpu:
         exampleFolder: bert
         exampleName: bert_ptq_cpu
-      resnet_ptq_cpu:
-        exampleFolder: resnet
-        exampleName: resnet_ptq_cpu
-      whisper:
-        exampleFolder: whisper
-        exampleName: whisper
-      resnet_vitis_ai_ptq_cpu:
-        exampleFolder: resnet
-        exampleName: resnet_vitis_ai_ptq_cpu
       bert_ptq_cpu_aml:
         exampleFolder: bert
         exampleName: bert_ptq_cpu_aml
       bert_ptq_cpu_docker:
         exampleFolder: bert
         exampleName: bert_ptq_cpu_docker
+      resnet_ptq_cpu:
```

Comment:
It seems the file is not changed. right?

Reply:
Yeah, looks like the tests were reordered in alphabetical order. But `cifar..` test is before `bert..`. Maybe @trajepl can update that too?

Reply:
Updated.